### PR TITLE
sample(05-sql-typeorm): added test e2e

### PR DESCRIPTION
**Pull request by Tony133**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Other... Please describe:

## What is the current behavior?

Issue Number: #1539

## What is the new behavior?
Added test e2e for sample `05-sql-typeorm`

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

## Other information
This PR, added test e2e for sample `05-sql-typeorm`

### CODE REVIEW

**Review by micalevisk — APPROVED** [review]

**Review by micalevisk — APPROVED** [review]

File: sample/05-sql-typeorm/test/users/users.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    isActive: true,
+  };
+
+  let app: INestApplication;
+
+  beforeAll(async () => {
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [
+        TypeOrmModule.forRoot({
+          type: 'mysql',
```
Comment (bassochette) [inline sample/05-sql-typeorm/test/users/users.e2e-spec.ts:22]: maybe you could use sqlite in memory database in order to run e2e tests without relying on a database to run?

Comment (Tony133) [reply sample/05-sql-typeorm/test/users/users.e2e-spec.ts:22]: I could also use sqlite, I usually use a test database with docker. Anyway let's see what they think of this e2e test, then I'll make the required changes.

Comment (bassochette) [reply sample/05-sql-typeorm/test/users/users.e2e-spec.ts:22]: anyway, LGTM
nice job comrade

**Review by bassochette — APPROVED** [review]

File: sample/05-sql-typeorm/test/users/users.e2e-spec.ts (hunk 1 above)
Comment (Tony133) [reply sample/05-sql-typeorm/test/users/users.e2e-spec.ts:22]: Thank you very much 👍, I am contributing to the NestJS framework from about three years now, I'm of home  😄

**Pull request merged**